arm/Microsoft.Network/connections/deploy.bicep

```diff
@@ -107,7 +107,7 @@ resource connection 'Microsoft.Network/connections@2021-02-01' = {
     enableBgp: enableBgp
     connectionType: virtualNetworkGatewayConnectionType
     routingWeight: routingWeight
-    sharedKey: vpnSharedKey
+    sharedKey: ((virtualNetworkGatewayConnectionType == 'ExpressRoute') ? vpnSharedKey : json('null'))
```

Comment:
Hey @jceval , just an optical comment: In bicep you can just write

```bicep
sharedKey: virtualNetworkGatewayConnectionType == 'ExpressRoute' ? vpnSharedKey : null
``` 

Thanks for the contribution 👍

Reply:
Thanks for the tip ✔️